src/components/Dashboard/Dashboard.module.scss

```diff
@@ -146,16 +132,45 @@
   }
 }
 
+.inputWrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  width: 100%;
+
+  @media (min-width: $screen-md-min) {
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 1rem;
+  }
+
+  label {
```

Comment:
Don't be lazy; create another class. Pass it as new param into the input as `labelClassName`. You can then avoid unexpected styling from some random over-generic selector. I wouldn't call `label`, which is this low in the tree as over-generic, but let's say we would add another label outside of the input, and due to the fact the style is in its wrapper, it would suddenly get random styles somebody who was not the original author might NOT expect.

Again, this is not over-generic. But I want to set the precedence :D

Reply:
@GLObus303 in this case i needed to reach label that is inside of the component textfield, is there better way?

Reply:
actually solved the issue other way by fixing set up of text field :)